tf-modules/archive/sf_event_sqs_to_db_records.tf

```diff
@@ -147,6 +147,7 @@ resource "aws_sqs_queue_policy" "sf_event_sqs_to_db_records_input_queue_policy"
 resource "aws_lambda_event_source_mapping" "sf_event_sqs_to_db_records_mapping" {
   event_source_arn = aws_sqs_queue.sf_event_sqs_to_db_records_input_queue.arn
   function_name    = aws_lambda_function.sf_event_sqs_to_db_records.arn
+  function_response_types = ["ReportBatchItemFailures"]
```

Comment:
Docs related to this are at https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html under `Implementing partial batch responses`